packages/terra-functional-testing/src/config/wdio.conf.js

```diff
@@ -129,7 +130,10 @@ exports.config = {
   // your test setup with almost no effort. Unlike plugins, they don't add new
   // commands. Instead, they hook themselves up into the test process.
   services: [
-    [TerraService],
+    [TerraService, {
+      /* Use to change the form factor (test viewport) used in the wdio run. */
+      formFactor: FORM_FACTOR || 'huge',
```

Comment:
Can you add some documentation for this option in the TerraService MDX file?

Reply:
Added doc. https://github.com/cerner/terra-toolkit/pull/438/commits/19c190061a6d0f428d0ab4b33b669bed04d9f219